### PR TITLE
[RFC] vim-patch:13d5aee,705ada1,298b440,5e9b2fa,7c764f7,681baaf,cbebd48

### PR DESCRIPTION
**Pull request by jamessan**
Update runtime files

https://github.com/vim/vim/commit/13d5aeef56e3140a8eb8f40c7062aa1c5700f76e

Ignored changes to
- doc/develop.txt, since they were all in the "Coding Style"
  section, which is completely different between Vim and Neovim.
- doc/tags, doc/todo.txt
- syntax/vim.vim, generated at build time

Update a few runtime files.

https://github.com/vim/vim/commit/705ada1aff27ecd9c47c690df817d043c2ceb5e2

Ignored changes to
- doc/eval.txt, all json related documentation
- doc/if_mzsch.txt
- doc/tags, generated at build time

Update runtime files.

https://github.com/vim/vim/commit/298b440930ecece38d6ea0505a3e582dc817e79b

Ignore changes to
- doc/netbeans.txt
- doc/tags: generated at build time
- doc/todo.txt
- doc/usr_41.txt, doc/various.txt: +channel related docs

Updated runtime files and translations.

https://github.com/vim/vim/commit/5e9b2fa9bb0e6061cf18457c173cd141a5dc9c92

Ignore changes to
- doc/tags: generated at build time
- doc/develop.txt, doc/todo.txt, doc/netbeans.txt, doc/vim-ja.UTF-8.1,
  doc/xxd-ja.UTF-8.1, lang/menu_*: Not applicable to Neovim
- doc/editing.txt: Crypt related
- doc/change.txt, doc/insert.txt, doc/various.txt: Removal of ex_extra
  tags, which already happened in Neovim
- doc/vim-ja.UTF-8.1, doc/xxd-ja.UTF-8.1

Make the python script executable.

https://github.com/vim/vim/commit/7c764f7bbf71a7a49baae07641efd2ead69e4d08

Update runtime files.

https://github.com/vim/vim/commit/681baaf4a4c81418693dcafb81421a8614832e91

Ignore changes to
- doc/channel.txt, doc/eval.txt, doc/usr_41.txt: Channel related docs
- doc/tags: Generated at build time
- doc/todo.txt: Not relevant to Neovim

Updated runtime files.

https://github.com/vim/vim/commit/cbebd4879cc78e670d79b2c57dc33d7b911c962a

Ignore changes to
- doc/channel.txt, doc/eval.txt: Channel related docs
- doc/tags: Generated at build time
- doc/todo.txt: Not relevant to Neovim
- ftplugin/man.vim: Change already present in autoload/man.vim

### CODE REVIEW

File: runtime/doc/eval.txt [hunk 1]
```diff
@@ -3665,7 +3668,8 @@ getftype({fname})					*getftype()*
 			getftype("/home")
 <		Note that a type such as "link" will only be returned on
 		systems that support it.  On some systems only "dir" and
-		"file" are returned.
+		"file" are returned.  On MS-Windows a symbolic link to a
+		directory returns "dir" instead of "link".
```
Comment (justinmk) [inline runtime/doc/eval.txt]: @jamessan This confirms what you asked in gitter a couple days ago (and what I observed in recent win32 gvim).

**Comment by justinmk** [comment]
LGTM. Will merge after build.

File: runtime/doc/eval.txt (hunk 1 above)
Comment (jamessan) [inline runtime/doc/eval.txt]: Yeah, that's actually why I was asking. :) I didn't have a Windows system handy to verify if the doc updates matched neovim's behavior (yet).

Comment (justinmk) [inline runtime/doc/eval.txt]: I only checked Vim, not Neovim. But it shouldn't hold this up, win32 nvim is still alpha.

**Comment by justinmk** [comment]
```
CMake Error at /home/travis/build/neovim/neovim/cmake/RunMsgfmt.cmake:8 (message):
  msgfmt failed to run correctly:
  /home/travis/build/neovim/neovim/src/nvim/po/ja.sjis.po:2869: duplicate
  message definition...

  /home/travis/build/neovim/neovim/src/nvim/po/ja.sjis.po:500: ...this is the
  location of the first definition

  /usr/bin/msgfmt: found 1 fatal error
```

**Comment by jamessan** [comment]
Fixed both duplicate messages.

**Pull request merged**